### ISSUE TITLE
Config Flow aborted: already configured

### ISSUE THREAD

**Issue by rhpijnacker** [JimStar/reolink_cctv#155]
After trying out the "official" Reolink component that now comes with HA, I quickly decided to move back.
However, when I try to re-add one of my cameras, I get an Unexpected Error:

```
2023-01-21 08:22:21.649 ERROR (MainThread) [custom_components.reolink_cctv.config_flow] Unexpected exception
Traceback (most recent call last):
  File "/config/custom_components/reolink_cctv/config_flow.py", line 76, in async_step_user
    self._abort_if_unique_id_configured()
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 1432, in _abort_if_unique_id_configured
    raise data_entry_flow.AbortFlow(error)
homeassistant.data_entry_flow.AbortFlow: Flow aborted: already_configured
```

I've tried removing the camera entries from .../config/.storage/core.config_entries,core, device_registry, entity_registry but that did not seem to make any difference.

I am running up-to-date Home Asssistant OS and reolink_cctv versions.

**Comment by JimStar** [comment]
Looks like you need to remove **all** entities created by the official integration to switch back to this one. It's not a right thing they just use the same device-ids al this initial integration they took as a source - so looks like now there is a conflict when you try to switch...  
Doesn't look like the problem of **this** project... it works absolutely OK before you try the official one..;)

**Comment by rhpijnacker** [comment]
Just to clarify: should I remove any Reolink (cctv) cameras I have via the UI, _then_ remove any reolink reference I can find in the .storage folder and then it should work?
Just trying to get things running again ...

**Comment by rhpijnacker** [comment]
I removed _all_ Reolink cameras from the UI.
Then I removed all devices and entities I could find from the .storage folder.
I rebooted just to be sure.

To my surprise, a camera entry for that camera that I cannot re-add was back!
However, from the details is is clear that this is the reolink_cctv entity that is restored, not the one from the "official" integration.

Is there some other place where this information is cached?
